ironfish/src/rpc/routes/wallet/types.ts

```diff
@@ -72,3 +75,41 @@ export async function getAssetBalanceDeltas(
 
   return assetBalanceDeltas
 }
+
```

Comment:
This probably belongs in a different file, it isn't a type.

Reply:
yeah, i'll add a separate `utils` module and then move the other non-type declarations from this file as well